vars/pushNewDockerImageTagChangePR.groovy

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/groovy
+import com.cloudbees.groovy.cps.NonCPS
+
+def call(body) {
+  // evaluate the body block, and collect configuration into the object
+  def config = [:]
+  body.resolveStrategy = Closure.DELEGATE_FIRST
+  body.delegate = config
+  body()
+
+  def flow = new io.fabric8.Fabric8Commands()
+
+  def dockerfileLocation = config.parentDockerfileLocation ?: 'Dockerfile.deploy'
+  def containerName = config.containerName ?: 'clients'
+  def autoMerge = config.autoMerge ?: false
+
+  for (int i = 0; i < config.projects.size(); i++) {
+    def project = config.projects[i]
+    def items = project.split('/')
+    def org = items[0]
+    def repo = items[1]
+    def dockerImage = config.propertyName
+    def tag = config.version
+    def id
+
+    stage "Updating ${project}"
+    sh "rm -rf ${repo}"
```

Comment:
This could be a bit dangerous in case someone called this function in the middle of doing something else in a workspace.  You could wrap the stage in a `ws{..}` which means it will have a unique location?